### PR TITLE
Update README for refining some description

### PR DESCRIPTION
**Pull request by chenliang613**
Update README for refining some description

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,22 +1,26 @@
...
-* CarbonData is designed to support very efficient compression and global encoding schemes,CarbonData can support actionable compression across data files for query engines to perform all processing on compressed/encoded data without having to convert the data. This improves the processing speed, especially for analytical queries. The data can be converted back to the user readable format just before returning the results to the user.
-* CarbonData is designed with various types of usecases in mind and provides flexible storage options. Data can be stored in completely columnar or row based formats or even in a mix of columnar and row based hybrid format.
-* CarbonData is designed to integrate into big data ecosystem, leverages Apache Hadoop,Apache Spark etc. for distributed query processing.
+Based on the below requirements, we investigated existing file formats in the Hadoop eco-system, but we could not find a suitable solution that satisfying requirements all at the same time, so we start designing CarbonData.
```
Comment (Vimal-Das) [inline README.md]: solution for satisfying

File: README.md [hunk 2]
```diff
@@ -1,22 +1,26 @@
...
+* Requirement5:Support HDFS so that customer can leverage existing Hadoop cluster. 
 
 ### Features
-* Fast analytic queries in seconds using built-in index, optimized for interactive OLAP-style query, high through put scan query, low latency point query. 
-* Fast data loading speed and support incremental load in period of minutes. 5 mins near realtime loading should be supported
-* Support concurrent query.
-* Support time based data retention. 
-* Supports SQL based query interface.
+CarbonData file format is a columnar store in HDFS, it has many features that a modern columnar format has, such as splittable, compression schema ,complex data type etc. And CarbonData has following unique features:
+* Stores data along with index: it can significantly accelerate query performance and reduces the I/O scans and CPU resources, where there are filters in the query.  CarbonData index is consisted of multiple level, a processing framework can leverage this index to reduce the task it needs to schedule and process, and it can also do skip scan in more finer grain unit (called blocklet) in task side scanning instead of scanning the whole file.
```
Comment (Vimal-Das) [inline README.md]: CarbonData index consists of

File: README.md [hunk 3]
```diff
@@ -1,22 +1,26 @@
...
-* Support concurrent query.
-* Support time based data retention. 
-* Supports SQL based query interface.
+CarbonData file format is a columnar store in HDFS, it has many features that a modern columnar format has, such as splittable, compression schema ,complex data type etc. And CarbonData has following unique features:
+* Stores data along with index: it can significantly accelerate query performance and reduces the I/O scans and CPU resources, where there are filters in the query.  CarbonData index is consisted of multiple level, a processing framework can leverage this index to reduce the task it needs to schedule and process, and it can also do skip scan in more finer grain unit (called blocklet) in task side scanning instead of scanning the whole file. 
+* Operable encoded data :Through supporting efficient compression and global encoding schemes, can query on compressed/encoded data, the data can be converted just before returning the results to the users, which is "late materialized". 
+* Column group: Allow multiple columns form a column group to store as row format, thus cost of column reconstructing is reduced.
```
Comment (Vimal-Das) [inline README.md]: Allow multiple columns to form a column group that would be stored as row format. This reduces the row reconstruction cost at query time.

**Comment by chenliang613** [comment]
Thanks for @Vimal-Das comments, updated it.

**Comment by jackylk** [comment]
LGTM

**Pull request merged**